src/middleware/cors/index.ts

```diff
@@ -12,7 +12,7 @@ type CORSOptions = {
 export const cors = (options?: CORSOptions): MiddlewareHandler => {
   const defaults: CORSOptions = {
     origin: '*',
-    allowMethods: ['GET', 'HEAD', 'PUT', 'POST', 'DELETE', 'PATCH'],
+    allowMethods: ['GET', 'HEAD'],
```

Comment:
I'm reconsidering. How about this line? I think the current implementation is OK.

```ts
['GET', 'HEAD', 'PUT', 'POST', 'DELETE', 'PATCH']
```

Indeed, it is more secure to limit `GET` and `HEAD`, but this is breaking changes. And it may be common to allow the above six methods by default. For example, Express cors implementation:

https://github.com/expressjs/cors/blob/f038e7722838fd83935674aa8c5bf452766741fb/lib/index.js#L10

Reply:
You are right, it would be a breaking change.

The most important part IMHO is not calling the `next` during the pre-flight request anyway.

Reply:
While leaving it like this is good to avoid a breaking change, I think it is imperative to indicate in the docs that the allowed HTTP methods are "all" by default.